### DISCUSSION TITLE
代码行高亮

### DISCUSSION THREAD

**Discussion by sixmillions** [imzbf/md-editor-rt#158] [Ideas]
```
\`\`\`ts {1,6-8}
import type { UserConfig } from "vuepress/cli";
import { defaultTheme } from "@vuepress/theme-default";

export const config: UserConfig = {
  title: "你好， VuePress",

  theme: defaultTheme({
    logo: "https://vuejs.org/images/logo.png",
  }),
};
\`\`\`
```

![Screenshot_2024-05-15-21-43-23-324_com.mmbox.xbrowser.pro.jpg](https://github.com/imzbf/md-editor-rt/assets/41658874/eb39858b-557f-4153-9b65-866f9af7d045)

你好，请问这个功能后续会有吗？

**Comment by imzbf** [comment]
这个之前在一个话题中讨论过，目前有现有的扩展，但是需要适配编辑器，后面我有时间会去研究一下，不过时间可能不会很快
